docs/content/doc/installation/from-source.en-us.md

```diff
@@ -106,6 +106,8 @@ To include assets, add the `bindata` tag:
 TAGS="bindata" make build
 ```
 
+WARNING: `generate` was dropped, `generate` may lost all the static files.
```

Comment:
Not sure what you mean here.

Reply:
@guillep2k I meand don't use `generate` any more because it will not include the static files.